rules/php56/tests/Rector/FunctionLike/AddDefaultValueForUndefinedVariableRector/Fixture/demo_file.php.inc

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Rector\Php56\Tests\Rector\FunctionLike\AddDefaultValueForUndefinedVariableRector\Fixture;
+
+final class DemoFile
+{
+    public function run()
+    {
+        $a['a'] = 'b';
+        $a['a'] = 'b';
+        $a['a'] = 'b';
+    }
+}
+?>
+-----
+<?php
+
+namespace Rector\Php56\Tests\Rector\FunctionLike\AddDefaultValueForUndefinedVariableRector\Fixture;
+
+final class DemoFile
+{
+    public function run()
+    {
+        $a = [];
```

Comment:
I think `$a = null` (existing behaviour) is fine, ref https://phpstan.org/r/a62c48b7-46ee-4d63-9847-46f7c434e90b

Reply:
`null` is fine, but I think that `[]` is more familiar

Reply:
The default value should be `[]`. It seems like a bug in Rector